buildpipeline/portable-linux.groovy

```diff
@@ -61,11 +61,13 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Ubuntu.1404.Amd64.Open',
                                      'Ubuntu.1604.Amd64.Open',
                                      'Ubuntu.1610.Amd64.Open',
-                                     // 'Ubuntu.1704.Amd64.Open',
                                      'suse.422.amd64.Open',
```

Comment:
Are the queue names case-sensitive? I ask because we seem to have inconsistent casing between the queue names.